Source/Container/HCMap_Internal.h

```diff
@@ -51,4 +52,9 @@ void HCMapPairDestroy(HCMapPairRef self);
 HCBoolean HCMapPairIsEqual(HCMapPairRef self, HCMapPairRef other);
 HCInteger HCMapPairHashValue(HCMapPairRef self);
 
+//----------------------------------------------------------------------------------------------------------------------------------
+// MARK: - Iteration
+//----------------------------------------------------------------------------------------------------------------------------------
+HCInteger _HCMapIteratorMinimumSizeRequiredForState(void);
```

Comment:
Does this have to be public for external access? Or is it just for testing that it is public? If it is just for testing, we can move it to the _Internal header and include it in the test case.

Reply:
Oh! This is already the internal header. Nice.